dotnet/src/dotnetcore/GxNetCoreStartup/Startup.cs

```diff
@@ -144,6 +144,10 @@ public void ConfigureServices(IServiceCollection services)
 			AppSettings settings = new AppSettings();
 			Config.ConfigRoot.GetSection("AppSettings").Bind(settings);
 
+			services.Configure<FormOptions>(options =>
+			{
+				options.MultipartBodyLengthLimit = settings.MaxFileUploadSize==0 ? DEFAULT_MAX_FILE_UPLOAD_SIZE_BYTES : settings.MaxFileUploadSize;
```

Comment:
This is also configurable from Environment Variables, right?

Reply:
This wasn't using that mechanism of Environment Variables (by mistake). Now it is fixed.